maquetado/scss/tech.scss

```diff
@@ -10,15 +19,15 @@
     border-radius: 8px;
     display: flex;
     flex-direction: column;
-    height: 300px;
+    height: 310px;
     margin: 40px 20px;
-    padding: 15px 10px 10px;
+    padding: 10px;
     width: 200px;
     position: relative;
     
     &::after {
         width: 200px;
-        height: 289px;
+        height: 300px;
```

Comment:
Why is this height different to the card height?

Reply:
I agree, why?